docs/yaml.md

```diff
@@ -106,3 +106,35 @@ For more details see the [OCI specification](https://github.com/opencontainers/r
 - `sysctl` sets a list of `sysctl` key value pairs that are set inside the container namespace.
 
 Further OCI values will be added, as the list is not yet complete.
+
+### Mount Options
+When mounting filesystem paths into a container - whether as part of `onboot`, `system` or `daemon` - there is a number of options of which you need to be aware. Using them properly is necessary for your containers to function properly.
```

Comment:
"of `onboot`, `system` or `daemon`" -> "of `onboot` or `services`"

"there is a number" -> "there are a number"